ember-bootstrap/types/ember-on-helper/helpers/on-document.d.ts

```diff
@@ -0,0 +1,16 @@
+import { HelperLike } from '@glint/template';
+
+declare module '@glint/environment-ember-loose/registry' {
+  export default interface Registry {
+    'on-document': HelperLike<{
+      Args: {
+        Positional: [
+          keyof DocumentEventMap,
+          (event: Event) => void,
```

Comment:
We can likely improve typing of the `event` to pick up the correct type based on the first argument from `DocumentEventMap`. But that's beyond my TypeScript knowledge. I would keep it as it is.

Cannot copy a solution from existing `{{on-window}}` typing. That one skips the event argument entirely.